SPECS/kernel-hci/kernel-hci.spec

```diff
@@ -68,15 +59,22 @@ BuildRequires:  libmspack-devel
 BuildRequires:  openssl
 BuildRequires:  openssl-devel
 BuildRequires:  pam-devel
+BuildRequires:  pciutils-devel
 BuildRequires:  procps-ng-devel
 BuildRequires:  python3-devel
 BuildRequires:  sed
-BuildRequires:  pciutils-devel
 Requires:       filesystem
 Requires:       kmod
 Requires(post): coreutils
 Requires(postun): coreutils
 ExclusiveArch:  x86_64
+# find_debuginfo.sh arguments are set by default in rpm's macros.
+# The default arguments regenerate the build-id for vmlinux in the
+# debuginfo package causing a mismatch with the build-id for vmlinuz in
+# the kernel package. Therefore, explicilty set the relevant default
+# settings to prevent this behavior.
+%undefine _unique_build_ids
+%undefine _unique_debug_names
```

Comment:
My guess is you moved these because of the linter. These are actually tied to the other globals

%global _missing_build_ids_terminate_build 1
%global _no_recompute_build_ids 1

Feel free to ignore the linter's recommendations for the kernel.